app/code/Magento/Elasticsearch/SearchAdapter/Query/Builder/MatchQuery.php

```diff
@@ -95,24 +97,26 @@ public function build(array $selectQuery, RequestQueryInterface $requestQuery, $
     }
 
     /**
-     * Prepare query.
+     * Prepare query
      *
      * @param string $queryValue
      * @param string $conditionType
      * @return array
      */
     protected function prepareQuery($queryValue, $conditionType)
```

Comment:
MHI fails because of protected functions:
![image](https://user-images.githubusercontent.com/1873745/128127285-a5d72307-bc63-4ad5-a779-c01f74f82e77.png)

As we discussed in Slack, people will have to fix their modules anyway, so it's better to get rid of these protected methods.

Please make them private and declared argument & return types

Reply:
Now it's fixed